include/odp/api/spec/traffic_mngr.h

```diff
@@ -497,6 +545,13 @@ typedef struct {
 	 * more of the marking API's. */
 	odp_bool_t marking_colors_needed[ODP_NUM_PACKET_COLORS];
 
+	/** The pkt_prio_mode indicates the packet priority mode, this TM system
+	 * needs to run with. A TM system may or may not support a given mode.
+	 * TM capabilities will indicate support for different modes by the
+	 * platform. Default value is ODP_TM_PKT_PRIO_MODE_PRESERVE.
+	 */
```

Comment:
I would like to propose the following shorter text:

Packet priority mode. TM capabilities indicate which modes are supported.
The default value is ODP_TM_PKT_PRIO_MODE_PRESERVE.